htlcswitch/switch.go

```diff
@@ -1378,8 +1395,15 @@ func (s *Switch) htlcForwarder() {
 	fwdEventTicker := time.NewTicker(15 * time.Second)
 	defer fwdEventTicker.Stop()
 
+out:
 	for {
 		select {
+		case blockEpoch, ok := <-s.blockEpochStream.Epochs:
```

Comment:
Is the reason we are not using`GetBestBlock()` performance?

Reply:
I believe so. In the Neutrino case, we end up going to disk. This also notifies us as soon as we know about the new block, rather than having to do some kind of long-polling with `GetBestBlock`.